src/renderer/components/+workloads-pods/details/volumes/variants/host-path.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import React from "react";
+import { DrawerItem } from "../../../../drawer";
+import type { VolumeVariantComponent } from "../variant-helpers";
+
+export const HostPath: VolumeVariantComponent<"hostPath"> = (
+  ({ variant: { path, type }}) => (
+    <>
+      <DrawerItem name="Node's Host Filesystem Path">
+        {path}
+      </DrawerItem>
+      <DrawerItem name="Check Behaviour">
```

Comment:
Why is `type` titled with "Check Behaviour"?

Reply:
Because that is what it means, here are the docs https://kubernetes.io/docs/concepts/storage/volumes/#hostpath

Reply:
I read that but didn't quite understand. Doesn't seem like the best description but you're right to match the docs. Only uses the word "check" for the empty string case. Getting the word "type" in there would be better I think. "Required Type"? "Type Check Behaviour"? Or leave it.